ansible/roles/build-yggdrasil-rpm/defaults/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+base_repo_dir: /workspace/workspace
+rpm_build_directory: ~/rpmbuild
+build_arch: x86_64
```

Comment:
Why are you building the worker for both archs, but yggdrasil for only one?

Reply:
OK for leaving it for later. But, you should be able to build an ARM target on an x86_64 system.

Reply:
Acknowledge. Eventually I think we will end up using what @eloycoto suggested and pull the latest rpms from copr's website:
https://copr.fedorainfracloud.org/coprs/jdzon/yggdrasil/
In essence, his suggestion probably renders this work obsolete, but I'm happy to throw it away for a much simpler solution if we decide to do so.
In the meanwhile I'll merge the PR once it's good to go :smile: